<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchMessage</class>
 <widget class="QWidget" name="SearchMessage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1180</width>
    <height>801</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>71</y>
     <width>1051</width>
     <height>581</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="playerLineEdit">
   <property name="geometry">
    <rect>
     <x>628</x>
     <y>20</y>
     <width>205</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="searchPlayer">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>20</y>
     <width>93</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>搜索球员</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="TeamLineEdit">
   <property name="geometry">
    <rect>
     <x>58</x>
     <y>20</y>
     <width>205</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="searchTeam">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>20</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>搜索球队</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
